libseccomp/src/notify.rs

```diff
@@ -24,7 +24,7 @@ fn get_errno() -> i32 {
 ///
 /// If both the libseccomp API level and the libseccomp version being used are
 /// less than 6 and 2.5.0, an error will be returned.
-fn notify_supported() -> Result<()> {
+pub fn notify_supported() -> Result<()> {
```

Comment:
I made this public so it could be used in the tests. Is this okay?

Reply:
> > ```diff
> > -fn notify_supported() -> Result<()> {
> > +pub fn notify_supported() -> Result<()> {
> > ```
>
> I made this public so it could be used in the tests. Is this okay?

If you add it to `CHANGELOG.md` I'm fine with it.